src/scss/includes/menu.scss

```diff
@@ -136,8 +136,8 @@ $productDrawerWidth: 744px;
 
     &-apps {
       position: absolute;
-      bottom: $spacing-m; 
-      right: $spacing-l; 
+      bottom: $spacing-m;
```

Comment:
Are changes in this file related to the scale stuff? Looks like a git add mistake.

Reply:
it seems like it's my linter that removed extra spaces at the end of each line. should I revert them?

Reply:
Sorry I hadn't seen it was just space removal on these two lines, but about the spacing change below and the added linebreaks? Looks like this file should be just removed from this PR.

Reply:
I'd rather keep it. In this ticket I moved the scale, attribution, zoom elements according to the new Design, and they're now offseted 16px from the right of the screen. If the menu button stays offseted 12px (spacing-s), it looks bad for me: 
![image](https://user-images.githubusercontent.com/1225909/118653498-e14a3f80-b7e7-11eb-832a-b69700410788.png)

Reply:
It is the point of shifting the zoom/geoloc slightly to the left: now they centered in the same column as the menu button.
![118653498-e14a3f80-b7e7-11eb-832a-b69700410788](https://user-images.githubusercontent.com/243653/118657072-6e42c800-b7eb-11eb-9fcc-1544f469ef09.png)

In any case, if it's not an explicit part of the issue about the scale, don't add this change here.